sandbox/src/App.js

```diff
@@ -125,6 +129,7 @@ function BasicExample() {
           <Route path="/redirectedNewPage" component={RedirectedNewPage} />
           <Route path="/identity" component={Identity} />
           <Route path="/configOverrides" component={ConfigOverrides} />
+          <Route path="/decisioningEngine" component={DecisioningEngine} />
```

Comment:
If renamed above, this route should also change to something like `/inAppMessages`